cluster/gce/gci/README.md

```diff
@@ -21,14 +21,63 @@ Policy](https://cloud.google.com/container-optimized-os/docs/resources/support-p
 Container-Optimized OS images are used by kubernetes [End-to-End tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) and
 [Node End-to-End tests](https://github.com/kubernetes/community/tree/master/contributors/devel/sig-node). To see current
 active releases, please refer to COS's [Release
-Notes](https://cloud.google.com/container-optimized-os/docs/release-notes). When
-choose images for testing, the latest LTS images is suggested for running
-presubmit, postsubmit or periodics tests since the image is stable and has
-latest bug and security fixes. For testing requiring new features or continuous integration,
-the latest dev or beta or stable channel can be used.
-are needed, the latest LTS images or stables are preferred.
+Notes](https://cloud.google.com/container-optimized-os/docs/release-notes).
+
+### How to choose an image in configuration file
+
+There are three ways to specify an image used by each testing suite: image,
+image_regex or image_family.
+
+  * image is preferred, but manual updates are needed to use a newly released
+    COS image, so the test suites don't use deprecated images. This will result
+    to frequent yaml configuration file update everytime COS releases new
+    image.One future option is to use an autobumper robot to update COS image
+    automatically.
+
+  * image_family should be used if you always want to use latest image in the
+    same family. Tests will start to use new images once COS releases
+    new image. This is not predictable and test can potentially be broken becase of this. The probability of a
+    breakage is low for LTS or stable image, but high for dev or beta image.
+    If things went wrong, it will be hard to rollback
+    images using image_regex and image_family.
+
+  * image_regex can also
+    be used if you want image with the same naming pattern. Latest image will be
+    chosen when multiple images match the regular expression. However, this
+    option is rarely seen in the test code.
+
+  * To update the images, using image in the same channel is preferred. Keep in
+    mind, 69 is the first LTS image. Before that, COS only has dev, beta and stable
+    images. That is why stable images are used quite frequently in current testing.
+    For now, images should slowly migrate from stable to LTS if possible. For
+    testing using dev or beta, we need to consider the original intention and
+    keep using image in existing channel unless we understand the underlying reason.
+
+### What image is needed for your test
+
+Consider the importance of tests and the stability of Container-Optimized OS, the
+following guidelines are proposed for image choice in E2E testing.
```

Comment:
I think it'll be helpful to also add examples of what the config may look like here in each case, especially for image_regex and family.

Reply:
Added for the first two. image_regex is never used, and maybe removed in future,